doc/README.rst

```diff
@@ -3,52 +3,60 @@
 The doc folder
 ==============
 
-The documentation for NEST is contained in this folder. We are currently working
-on improving the documentation and making updates. However, if you notice
-something out of place, you can `submit an issue <https://nest.github.io/nest-simulator/development_workflow#reporting-bugs-and-issues>`_
-in our GitHub repository.
+This folder contains the static parts of the documentation for NEST,
+as well as the infrastructure and tools for extracting documentation
+dynamically from the source code and building a unified HTML site
+using Sphinx.
 
-Build the docs locally
------------------------
+For each of the releases, the fully built documentation is available
+at https://nest-simulator.readthedocs.io/
+
+Build the user documentation locally
+------------------------------------
 
-You will need the NEST source code in your project::
+If you intend to work on the documentation, or if you want to obtain a
+local version of it for other reasons, you can build the documentation
+by simply running
 
+::
 
-    cd nest-simulator/doc/
+    make html
 
-Requirements for building the docs include::
+in the build directory of NEST (i.e. the directory where you ran
+``cmake``). To install the dependencies for building the
+documentation, you can either directly run
 
- python3
- setuptools
- pandoc
- sphinx
- recommonmark
- sphinx-rtd-theme
- nbsphinx
- breathe
- sphinx-gallery
- ipython
- mock
+::
 
+    pip install -r <nest_source_dir>/doc/requirements.txt
 
-You can now build the docs using this command (make sure you are in the doc folder)::
+or alternatively install the dependencies into a conda environment by
+running
 
-    make html
+::
 
+    conda env create -f <nest_source_dir>/doc/environment.yml
+    conda activate conda/
 
-Once this complete, you can open `_build/html/index.html` to view the docs!
+Once this complete, you can open `doc/html/index.html` to view the docs.
 
 ::
 
-    xdg-open _build/html/index.html
+    xdg-open doc/html/index.html
 
+Developer documentation
+-----------------------
+    
 For **developer documentation**, we use `doxygen <http://doxygen.org/>`__
 comments extensively throughout NEST.
 After installing NEST, you can extract comments from the source code with
 ``make doc`` and a doxygen folder with html files will be generated in the doc
```

Comment:
Should we rename that target to something like `make dev-doc`?

Might also want to point out where the doc will be located.

Reply:
Yes, we should. However, I'd like to leave that to #1905.